fontforge/scripting.c

```diff
@@ -2374,8 +2375,6 @@ static void bExport(Context *c) {
 	if ( c->curfv->selected[i] && (gid=c->curfv->map->map[i])!=-1 &&
 		SCWorthOutputting(c->curfv->sf->glyphs[gid]) )
 	    ScriptExport(c->curfv->sf,bdf,format,gid,format_spec,c->curfv->map);
```

Comment:
Right. But if `format_spec = pt` and `tmp = pt` and `free(tmp)` happens before this line, ...